services/drafts.ts

```diff
@@ -1,19 +1,18 @@
 import { getPool } from '../database';
 import { sql} from 'slonik';
 import {update as slonikUpdate} from 'slonik-utilities';
-import {Draft, DraftBody, DraftUpdate, PendingDraft} from '../types/draft';
-import {filterNullValues} from "../helpers";
-import {number} from "zod";
+import {Draft, DraftBody, DraftResponse, DraftUpdate, PendingDraft} from '../types/draft';
+import {filterNullValues, countSchema} from "../helpers";
 import { faker } from "@faker-js/faker";
 
-async function index(email: string, type?: string, cursor?: number, limit?: number): Promise<readonly Draft[]> {
+async function index(email: string, type?: string, cursor?: number, limit?: number): Promise<readonly DraftResponse[]> {
```

Comment:
suggestion/ refactor: 4 params is more than I generally like in a function. 3 is my limit, 2 is better, 1 is best (sometimes none ;) ) . 
In these situations what I like to do is define a type / parameter object. i.e. 
```ts
interface VoteListParams = {
  email: string,
  type?: string,
  cursor?: number,
  limit?: number
}

async function index({email, type, cursor, limit}: VoteListParams): Promise<readonly DraftResponse[]>
{...}
```
I think it's more readable/maintainable.